### DISCUSSION TITLE
how to run directus on domain subdirectory

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#11223] [Archive]
My react js project running is on https://www.example.com on AWS. i want to run my directus backend like https://www.example.com/directus. how can i do this?

**Comment by azrikahar** [comment]
This technically depends on how your React project is actually setup/hosted. Generally a reverse proxy is one of the approach used in the scenario you described. You can read my comment on another similar question over here: https://github.com/directus/directus/discussions/10159#discussioncomment-1721975

Can't say I'm too familiar with AWS solutions personally, but lets say if your React project is hosted on AWS Amplify or even S3 for example, with Directus on an EC2 or ECS, you may need to use solutions such as [Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#path-conditions) or perhaps the [other ways outlined in this AWS blog post](https://aws.amazon.com/blogs/architecture/serving-content-using-fully-managed-reverse-proxy-architecture/).

**Reply by rijkvanzanten** [reply]
It really really depends on what services you use in AWS. There's at least a dozen different ways to setup the networking for something like this. If you're using an application load balancer in ELB, you could setup a path-style target group and have it redirect to the IP and Port of where you're running Directus (be it an ECS service, EC2 VPS, etc)

**Comment by techtroves** [comment]
Kindly refer
https://github.com/directus/directus/discussions/10159#discussioncomment-10513408
